querier.go

```diff
@@ -266,33 +266,86 @@ func (q *blockQuerier) Close() error {
 // checked to not exist in series the postings list points to.
 // It returns EmptyPostings() if it can be determined beforehand that no results will be found.
 func PostingsForMatchers(ix IndexReader, ms ...labels.Matcher) (index.Postings, error) {
-	var its []index.Postings
+	var its, notIts []index.Postings
```

Comment:
Oh my... will need to adapt https://github.com/improbable-eng/thanos/blob/5b74df2ae745e61f1ba09c347e63dea271df6d2e/pkg/store/bucket.go#L1223 for this to get this into Thanos (:

Reply:
https://github.com/improbable-eng/thanos/issues/1205